manifests/params_manager.pp

```diff
@@ -178,14 +178,13 @@
       $vulnerability_detector_provider_debian                 = 'yes'
       $vulnerability_detector_provider_debian_enabled         = 'no'
       $vulnerability_detector_provider_debian_os              = ['wheezy',
-        'stretch',
```

Comment:
For `canonical`, `focal` and `jammy` are missing (line 173)

Reply:
dome